chia/full_node/mempool.py

```diff
@@ -17,6 +18,18 @@
 from chia.util.ints import uint64
 
 
+class MempoolRemoveReason(IntEnum):
```

Comment:
this looks just like an enum. does this have any benefits over an enum?

Reply:
Nope! I just got in the habit of using IntEnum from us using Streamable to manage on-the-wire values. I'll change that to Enum.